### PR TITLE
[Tizen.Applications.PackageManager][TCSACR-272][Add] Add Dependency Info APIs

### PR DESCRIPTION
**Pull request by ilho159kim**
Add api for get dependency from package and package archive

### CODE REVIEW

File: src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs [hunk 1]
```diff
@@ -48,6 +48,8 @@ public class Package
 
         private Dictionary<IntPtr, Interop.PackageManager.PackageManagerSizeInfoCallback> _packageManagerSizeInfoCallbackDict = new Dictionary<IntPtr, Interop.PackageManager.PackageManagerSizeInfoCallback>();
         private int _callbackId = 0;
+        private List<PackageDependencyInformation> _dependency_to;
```
Comment (myroot) [inline src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs]:
```suggestion
        private List<PackageDependencyInformation> _dependencyTo;
```

File: src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs [hunk 2]
```diff
@@ -48,6 +48,8 @@ public class Package
...
+        private List<PackageDependencyInformation> _dependency_from;
```
Comment (myroot) [inline src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs]:
```suggestion
        private List<PackageDependencyInformation> _dependencyFrom;
```

File: src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs [hunk 3]
```diff
@@ -216,6 +218,18 @@ public IEnumerable<ApplicationInfo> GetApplications(ApplicationComponentType com
             return GetApplications(ToApplicationType(componentType));
         }
 
+        /// <summary>
+        /// Requested privilege for the package.
```
Comment (myroot) [inline src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs]: Wrong description

File: src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs [hunk 4]
```diff
@@ -216,6 +218,18 @@ public IEnumerable<ApplicationInfo> GetApplications(ApplicationComponentType com
...
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public IEnumerable<PackageDependencyInformation> DependencyTo { get { return _dependency_to; } }
+
+        /// <summary>
+        /// Requested privilege for the package.
```
Comment (myroot) [inline src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs]: Wrong description

File: src/Tizen.Applications.PackageManager/Tizen.Applications/PackageDependencyInformation.cs [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    /// <summary>
+    /// This class has read-only properties to get the package dependency information.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class PackageDependencyInformation
+    {
+        private string _from;
+        private string _to;
+        private string _type;
+        private string _required_version;
```
Comment (myroot) [inline src/Tizen.Applications.PackageManager/Tizen.Applications/PackageDependencyInformation.cs]:
```suggestion
        private string _requiredVersion;
```

File: src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs (hunk 1 above)
Comment (ilho159kim) [reply src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs]: I fixed that

File: src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs (hunk 2 above)
Comment (ilho159kim) [reply src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs]: _dependency

File: src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs (hunk 3 above)
Comment (ilho159kim) [reply src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs]: I fixed this description

File: src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs (hunk 4 above)
Comment (ilho159kim) [reply src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs]: I fixed this description

File: src/Tizen.Applications.PackageManager/Tizen.Applications/PackageDependencyInformation.cs (hunk 5 above)
Comment (ilho159kim) [reply src/Tizen.Applications.PackageManager/Tizen.Applications/PackageDependencyInformation.cs]: I fixed that

File: src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs [hunk 6]
```diff
@@ -216,6 +218,18 @@ public IEnumerable<ApplicationInfo> GetApplications(ApplicationComponentType com
             return GetApplications(ToApplicationType(componentType));
         }
 
+        /// <summary>
+        /// Packages that this package is required.
```
Comment (journey2w) [inline src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs]: Suggestion:
	        /// Packages required by this package.

**Pull request merged**